source/MRMesh/MRCNCMachineSettings.h

```diff
@@ -35,12 +42,17 @@ class CNCMachineSettings
     void setHomePosition( const Vector3f& homePosition ) { homePosition_ = homePosition; }
     const Vector3f& getHomePosition() const { return homePosition_; }
 
+    MRMESH_API bool operator==( const CNCMachineSettings& rhs );
```

Comment:
= default?

Reply:
we contain all settings for all (3) axes, but can do not using some axis, so that we don't need compare this axis